DeleteButton.js

```diff
@@ -0,0 +1,19 @@
+export class DeleteButton {
+    constructor(iD, userContainer) {
+        this.url = 'https://jsonplaceholder.typicode.com/users'
+        this.iD = iD + 1;
+        console.log(this.iD)
+        const deleteButton = document.createElement("button");
+        deleteButton.classList.add('delete-button');
+        deleteButton.innerHTML = "DELETE"
+        userContainer.append(deleteButton);
+        deleteButton.addEventListener("click", this.deleteUser)
+    }
+
+    deleteUser = () => {
+        const deleteUser =  fetch(this.url + '/' + this.iD, {
```

Comment:
use string literal instead of string concatenation

Reply:
you are assigning the promise returned from fetch to the `const deleteUser`, but the promise is never awaited, and you have no control over if the backend returned an Error or a Sucess.

Reply:
1st done

Reply:
2nd done but im not sure